Geometry/DTGeometryBuilder/src/DTGeometryParsFromDD.cc

```diff
@@ -18,18 +21,25 @@
 #include "DataFormats/MuonDetId/interface/DTChamberId.h"
 #include "DataFormats/GeometrySurface/interface/RectangularPlaneBounds.h"
 #include "DataFormats/Math/interface/GeantUnits.h"
-
+#include "DataFormats/GeometryVector/interface/Basic3DVector.h"
+#include "CLHEP/Units/GlobalSystemOfUnits.h"
```

Comment:
Delete `GlobalSystemOfUnits.h`. It should not be used. I'm not sure what it is needed for in the file, but `GeantUnits.h` should provide the same functionaliy.

Reply:
@cvuosalo : done